docs/design/autoware-interfaces/components/localization.md

```diff
@@ -0,0 +1,77 @@
+# Localization
+
+![Node diagram](images/Localization-Bus-ODD-Architecture.drawio.svg)
```

Comment:
@xmfcx Did you recommend using Mermaid for component diagrams before? I don't remember that.

Reply:
@kenji-miyake I've created this diagram in draw.io (diagrams.net) https://drive.google.com/file/d/1amCZ8sxCuY1isEVhGw99oKeIoqfV25Po/view?usp=sharing

So this was taken from there. But I am open to making these in Mermaid too.

Reply:
I see, thank you.
I think draw.io is also acceptable for now.